.github/workflows/build_librealsense_ROS2_Ubuntu.yaml

```diff
@@ -0,0 +1,60 @@
+name: Build_ROS2
+
+on:
+  push:
+    branches: ['**']
+  pull_request:
+    branches: ['**']
+
+jobs:
+
+  test_docker:
+    runs-on: ubuntu-latest
+    strategy:
```

Comment:
Why is that empty key needed?

Reply:
According to a [sample ](https://github.com/ros-tooling/setup-ros#:~:text=%24%7B%7B%20matrix.os%20%7D%7D-,strategy,-%3A%0A%20%20%20%20%20%20matrix%3A) of _ros-tooling/setup-ros_ I think that is key of matrix key...
I will remove the key and check how it will work without it.

Reply:
I understand I was confused because of the added empty line
No need to remove, I thought it is a parallel key with no value, but it is a parent..